### PR TITLE
docs: integrate README and FAQ

### PR DESCRIPTION
**Pull request by gbartolini**
Minor changes to the main README and FAQ (which now includes a list of other operators for PostgreSQL).

### CODE REVIEW

File: docs/src/faq.md [hunk 1]
```diff
@@ -204,14 +204,45 @@ of truth to:
...
+**Should I manually resync a former primary with the new one following a
+failover?**
+
+No. The operator does that automatically for you, and relies on `pg_rewind` to
+synchronize the former primary with the new one.
+
+**Are there any other operators for Postgres out there?**
+
+Yes, of course. And our advice is that you look at all of them and compare
+them with CloudNativePG before forming your decision. You will see that
```
Comment (phisco) [inline docs/src/faq.md:216]:
```suggestion
them with CloudNativePG before taking your decision. You will see that
```

File: docs/src/faq.md [hunk 2]
```diff
@@ -204,14 +204,45 @@ of truth to:
...
+most of these operators use an external failover management tool (Patroni
+or similar) and rely on stateful sets.
```
Comment (phisco) [inline docs/src/faq.md:218]:
```suggestion
or similar) and rely on StatefulSets.
```

**Review by phisco — COMMENTED** [review]
Few minor nits

File: docs/src/faq.md [hunk 3]
```diff
@@ -365,9 +396,9 @@ Postgres instance, with dedicated storage.
 We proved that this extreme architectural pattern works when we benchmarked
 [running PostgreSQL on bare metal Kubernetes with local persistent
 volumes](https://www.2ndquadrant.com/en/blog/local-persistent-volumes-and-postgresql-usage-in-kubernetes/).
-A current limitation of CloudNativePG, which will be overcome in future
-releases, is the lack of support for tablespaces so that horizontal
-partitioning can be easily implemented.
+A current limitation of CloudNativePG, which will be overcome in version 1.22,
+is the lack of support for tablespaces so that horizontal partitioning can be
+easily implemented.
```
Comment (jsilvela) [inline docs/src/faq.md:401]:
```suggestion
is the lack of support for tablespaces. Once tablespaces are available, horizontal partitioning can be
easily implemented.
```

**Review by jsilvela — APPROVED** [review]

File: ADOPTERS.md [hunk 4]
```diff
@@ -33,4 +33,4 @@ This list is sorted in chronological order, based on the submission date.
 | [Tembo](https://tembo.io/) | @tembo-io | 2023-07-17 | Tembo is the developer platform for PostgreSQL extensions. Build and share extensions with [Trunk](https://pgt.dev), and use any extension on Tembo Cloud. |
 | [CNDI](https://cndi.dev) | @johnstonmatt | 2023-08-21 | Provides simple workflow to deploy self-hosted CloudNativePG clusters with GitOps and Infrastructure as Code. |
 |[PITS Global Data Recovery Services](https://www.pitsdatarecovery.net/)| @benjx1990 | 2023-09-07 | CloudNativePG is  used to easily manage highly-loaded database clusters |
```
Comment (mnencia) [inline ADOPTERS.md:35]:
```suggestion
| [PITS Global Data Recovery Services](https://www.pitsdatarecovery.net/) | @benjx1990 | 2023-09-07 | CloudNativePG is  used to easily manage highly-loaded database clusters |
```

**Review by mnencia — APPROVED** [review]

**Review by sxd — APPROVED** [review]

**Pull request merged**